device/accton/x86_64-accton_as5812_54x-r0/Accton-AS5812-54X/port_config.ini

```diff
@@ -0,0 +1,73 @@
+# name         lanes              alias         index
+Ethernet0      13                 tenGigE0      0
```

Comment:
why does the index start from 0? are your front panel port numbering start from 0?

Reply:
No, the port number on the panel starts from 1. 
Most port_config.ini of accton devices starts from Ethernet0,  I just follow the way we used to be.

Reply:
check here. https://github.com/Azure/SONiC/wiki/Porting-Guide

index: Physical index of port (optional - needed if ports do not begin with index zero or increment by one), When this column is missing the index starts from 0 and increase by 1 for each row. It is recommend to explicitly set this index column and make the index number match the labeling of the front panel ports. AKA: if the front panel ports are starting from 1, the index should start from 1. If the front panel ports start from 0, the index should start form 0. The break out ports from the same physical port should have same index.